exams/pearson/constants.py

```diff
@@ -16,3 +17,9 @@
     "EXAMS_SFTP_PASSWORD",
     "EXAMS_SFTP_UPLOAD_DIR",
 ]
+
+# Custom csv dialect for Pearson
+csv.register_dialect(
+    PEARSON_CSV_DIALECT,
+    delimiter='\t',
+)
```

Comment:
I was, I'm not a huge fan of how Python implements dialects in global state like this, but that's how the API is designed :-1: . Since we're only configuring 1 setting now and it can be passed to the writer/writer constructors instead I've refactored this to use an options `dict` instead to avoid the global code.